server/routes/activitiesMigration/activitiesMigrationController.ts

```diff
@@ -358,6 +358,9 @@ export default class ActivitiesMigrationController {
       | where name endswith 'ignored'
       | join kind=leftouter (
         traces
+        | where timestamp between (datetime(${ActivitiesMigrationController.toISODateTime(
+          startedDate,
+        )}) .. datetime(${ActivitiesMigrationController.toISODateTime(endedDate)}))
```

Comment:
Testing on preprod showed that the queries were too slow (presumably due to increased volumes). Limiting the time range of the joined table helped a lot.